guides/source/upgrading_ruby_on_rails.md

```diff
@@ -257,6 +257,35 @@ See the [Testing Rails Applications](https://guides.rubyonrails.org/testing.html
 
 If you run a single file's tests (`bin/rails test test/models/user_test.rb`), `test:prepare` will not run before it.
 
+### Import syntax from `@rails/ujs` is modified
+
+Starting from Rails 7.1, the syntax for importing modules from `@rails/ujs` is modified.
+Rails no longer supports the direct import
+of a module from `@rails/ujs`.
+If users create and build a JavaScript file
+with the line `import { fileInputSelector } from "@rails/ujs"`,
+they will encounter the error `export 'fileInputSelector'
+(imported as 'fileInputSelector')
+was not found in '@rails/ujs' (possible exports: default)`.
+
+In Rails 7.1, users should first import the Rails object directly from `@rails/ujs`.
+Users can then import specific modules
+from the Rails object.
+
+An example of imports in Rails 7.1 is shown below:
+
+```javascript
+import Rails from "@rails/ujs"
+ // alias the method
+const fileInputSelector = Rails.fileInputSelector
+// reference it from the Rails object where it is used
+Rails.fileInputSelector(...)
+```
+
+
+
+
+
```

Comment:
```suggestion
```

Reply:
Thank you very much for the suggestions. I have implemented these suggested changes in the updated PR.